src/guc.c

```diff
@@ -180,6 +181,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable("timescaledb.enable_contagg_reorder_groupby",
```

Comment:
maybe this should be enable_cagg_reorder_groupby, isn't that the abbreviation we usually use for continuous aggs